test/integ_tests/pulse_control/1_Bringup_experiments_mocks.py

```diff
@@ -0,0 +1,22 @@
+
+
```

Comment:
I could update a notebook such that it no longer works against the service, but the existing test continues to pass. (For instance, pass in the wrong results from one call to the next. This won't be caught since these tests don't perform any assertions). How would we catch such regressions?

Reply:
There is a space for correctness testing. Each mock file is called with
```
def post_run(tb):
```
Where we should put assertions for correctness testing. Needless to say, that will be a lot of work, and will be outside the scope of this PR.